app.config.ts

```diff
@@ -90,5 +91,7 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
     "@react-native-firebase/app",
     "@react-native-firebase/crashlytics",
     ["expo-build-properties", { ios: { useFrameworks: "static" } }],
+    ["expo-updates", { username: "infinitered" }],
```

Comment:
did we determine if we needed this AND the checkAutomatically flag? I mean it works so can leave it

Reply:
yeah not 100% sure. don't want to mess with it since it works lol